source/index.html

```diff
@@ -2,144 +2,180 @@
+              </picture>
             </div>
           </li>
-          <div class="small_item__wrapper">
-            <li class="advantages__item small_item">
-              <h2 class="advantages__title">НЕБОЛЬШАЯ ПЛОЩАДЬ</h2>
+          <li class="advantages__wrapper__item-small">
+            <li class="advantages__item advantages__item--small">
+              <h2 class="advantages__title">Небольшая площадь</h2>
               <p class="advantages__number">— №3—</p>
-              <p class="advantages__content">ВСЕ интересные места находятся очень близко</p>
+              <p class="advantages__content">Все интересные места находятся очень близко</p>
             </li>
-            <li class="advantages__item small_item">
-              <h2 class="advantages__title">КРАСИВАЯ ДОРОГА</h2>
+            <li class="advantages__item advantages__item--small">
+              <h2 class="advantages__title">Красивая дорога</h2>
               <p class="advantages__number">— №4 —</p>
-              <p class="advantages__content">ЕХАТЬ В СЕДОНУ ИЗ ЛАС-ВЕГАСА совсем НЕ СКУЧНО!</p>
+              <p class="advantages__content">Ехать в Седону из Лас-Вегаса совсем не скучно!</p>
             </li>
-            <li class="advantages__item small_item">
-              <h2 class="advantages__title">МАЛО ТУРИСТОВ</h2>
+            <li class="advantages__item advantages__item--small">
+              <h2 class="advantages__title">Мало туристов</h2>
               <p class="advantages__number">— №5 —</p>
               <p class="advantages__content">Большинство едет в гранд каньони толпится там</p>
             </li>
-          </div>
+          </li>
         </ul>
       </section>
-      <section class="main-form main-form__wrapper">
-        <p class="main-form__title">ЗАИНТЕРЕСОВАЛИСЬ?</p>
-        <p class="main-form__info">Укажите предполагаемые датыпоездки, и мы покажем вам лучшиепредложения гостиниц в седоне</p>
-        <button class="main-form__button">Найти гостиницу</button>
+      <section class="page-main__form">
+        <p class="page-main__form__title">Заинтересовались?</p>
+        <p class="page-main__form__info">Укажите предполагаемые датыпоездки, и мы покажем вам лучшиепредложения гостиниц в седоне</p>
+        <a class="page-main__button" href="form.html">Найти гостиницу</a>
+      </section>
+      <section class="page-main__map">
+        <h3 class="visually-hidden">интерактивная карта</h3>
+        <picture>
+          <source media="(min-width:1200px)" srcset="img/map-desktop@1x.jpg 1x, img/map-desktop@2x.jpg 2x">
+          <source media="(min-width:768px)" srcset="img/map-tablet@1x.jpg 1x, img/map-tablet@2x.jpg 2x,">
+          <img class="page-main__map__img" src="img/map-mobile@1x.jpg" srcset= "img/map-mobile@2x.jpg 2x" width="320" height="350"></img>
```

Comment:
```suggestion
          <img class="page-main__map__img" src="img/map-mobile@1x.jpg" srcset= "img/map-mobile@2x.jpg 2x" width="320" height="350">
```
У `img` не должно быть закрывающего тега.